SU2_CFD/include/fluid/CFluidFlamelet.hpp

```diff
@@ -34,17 +34,19 @@
 #define USE_MLPCPP
 #endif
 #include "CFluidModel.hpp"
-
+#include <memory>
 class CFluidFlamelet final : public CFluidModel {
  private:
   ENUM_DATADRIVEN_METHOD Kind_DataDriven_Method =
       ENUM_DATADRIVEN_METHOD::LUT; /*!< \brief Interpolation method for data set evaluation. */
 
   enum LOOKUP_TD { TEMPERATURE, HEATCAPACITY, VISCOSITY, CONDUCTIVITY, DIFFUSIONCOEFFICIENT, MOLARWEIGHT, SIZE };
 
+  su2double beta_progvar, beta_enth_thermal, beta_enth, beta_mixfrac;
   int rank;
 
-  bool include_mixture_fraction = false; /*!< \brief include mixture fraction in controlling variables. */
+  bool include_mixture_fraction = false, /*!< \brief include mixture fraction in controlling variables. */
+      PreferentialDiffusion = false;     /*!< \brief use preferential diffusion physics. */
```

Comment:
It would be pleasant to keep the style consistent,
```suggestion
       preferential_diffusion = false;     /*!< \brief use preferential diffusion physics. */
```